src/security/x-powered-by.conf

```diff
@@ -18,6 +18,11 @@
 #
 # https://php.net/manual/en/ini.core.php#ini.expose-php
 
+
+# Note you need both below as the "always" option doesn't always work with some servers for some reasons.
 <IfModule mod_headers.c>
-    Header unset X-Powered-By
+    Header always unset X-Powered-By
+    Header unset X-Powered-By  
```

Comment:
Hmm, I prefer we stick to only one directive.
Not sure about the JBoss case, I can't find other references where the issue is mentioned, so the directive with the `always` condition is sufficient.

Reply:
Sure, I'm really no expert in this.